src/components/badge/badge.scss

```diff
@@ -13,7 +13,7 @@
 .badge-container {
     display: inline-block;
     font-size: 0.7rem;
-    min-width: 1.2rem;
+    min-width: .6rem;
```

Comment:
Small sidenote: I don't think this commit should be a `refactor`, it's either a `feat` or a `fix` 🙂

`refactor` is for changes to the code that has no impact on how the code functions (including no visual impact). Commits of type `refactor` will not be listed in the changelog.